src/server/src/rollbar/rollbar-exception.filter.ts

```diff
@@ -28,7 +28,10 @@ export class RollbarExceptionFilter extends BaseExceptionFilter {
   }
 
   catch(exception: unknown, host: ArgumentsHost): void {
-    if (exception instanceof HttpException && !isWhitelisted(exception)) {
+    if (
+      (exception instanceof HttpException && !isWhitelisted(exception)) ||
+      exception instanceof Error
+    ) {
```

Comment:
Since `HttpException` is a subclass of `Error` this meant that our whitelist wasn't doing anything.

To check this, I set a breakpoint here and stepped through execution here after checking the `/healthcheck/` endpoint while the TopoJSON data was still loading.

This fixed it:
```suggestion
    if (
      (exception instanceof HttpException && !isWhitelisted(exception)) ||
      (exception instanceof Error && !(exception instanceof HttpException))
    ) {
```